### PR TITLE
docs(readme): add Exthouse - ext perf analyzer

### PR DESCRIPTION
**Pull request by denar90**
**Summary**

Adds [Exthouse](https://github.com/treosh/exthouse) - web extension analyzer that provides a report about web extension impact on web performance.

**Quick overview**:

Screenshot of Grammarly extension performance report generated by Exthouse

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -306,6 +306,7 @@ This section details services that have integrated Lighthouse data. If you're wo
 ## Related Projects
 Other awesome open source projects that use Lighthouse.
 
+* **[Exthouse](https://github.com/treosh/exthouse)** - Analyze the impact of a browser extension on web performance.
```
Comment (patrickhulce) [inline readme.md]: just waiting for the day someone names their project "AAA Lighthouse" to take over that prime real estate 😆

Comment (denar90) [reply readme.md]: 😂

Comment (exterkamp) [reply readme.md]: brb making `___aaaa___a1111_Lighthouse_Analyzer`

Comment (connorjclark) [reply readme.md]: `<NUL>BestAppEver` wins

**Comment by denar90** [comment]
Thank you folks :)

**Pull request merged**